.werft/build.ts

```diff
@@ -396,6 +397,12 @@ export async function deployToDev(deploymentConfig: DeploymentConfig, workspaceF
         let flags = commonFlags
         flags += ` --set components.wsDaemon.servicePort=${wsdaemonPortMeta}`;
         flags += ` --set components.registryFacade.ports.registry.servicePort=${registryNodePortMeta}`;
+
+        let nodeAffinityValues = [
```

Comment:
We typically use `const` for local "variables" that we do not intend to alter.

Reply:
Done